src/sentry/utils/performance_issues/detectors/consecutive_db_detector.py

```diff
@@ -216,7 +216,7 @@ def is_event_eligible(cls, event, project: Project = None) -> bool:
 
         if request:
             url = request.get("url", "") or ""
-            method = request.get("method", "") or ""
+            method = request.get("http.method") or request.get("method", "") or ""
```

Comment:
```suggestion
            method = request.get("http.method", "") or request.get("method", "") or ""
```

m: Should we also guard for this here? Prob also requires us to add a quick sanity check unit test as well then.